source/index.html.md

```diff
@@ -492,7 +510,7 @@ https://ns-api-staging.betable.com/lfs/v100\
 This method is used for closing a Round. A round can consist of zero or more Wagers, and zero or more Payouts. After a round is closed then no additional Wagers or Payouts can be associated to the Round.
 
 <aside class="warning">
-All <code>authorized</code> Wagers and Payouts must be resolved (ie. settled or void) before the Round can be closed.
```

Comment:
I deleted this but it seemed intentional

Reply:
Yes it is intentional, but I guess it is spelled out right after that it must be resolved so that is fine.  Also it is in a warning label so all of this should highlight the nuance.

Reply:
sorry i remove the `<code>authorized</code>` section specifically